waiter/integration/waiter/kubernetes_scheduler_integration_test.clj

```diff
@@ -40,6 +40,20 @@
             (is (<= initial-rs-snapshot-version rs-snapshot-version'))
             (is (< rs-snapshot-version' rs-watch-version'))))))))
 
+; test that we can provide a custom docker image that contains /tmp/index.html with "Integration Test Image" in it
+(deftest ^:parallel ^:integration-slow test-kubernetes-custom-image
+  (testing-using-waiter-url
+    (when (using-k8s? waiter-url)
+      (let [{:keys [body]} (make-kitchen-request
+                             waiter-url
+                             {:x-waiter-name (rand-name)
+                              :x-waiter-image (System/getenv "INTEGRATION_TEST_CUSTOM_IMAGE")
```

Comment:
What happens if this is not set? It seems like we should have an assertion for that.